### ISSUE TITLE
Why doesn't Topola-viewer display the married name?

### ISSUE THREAD

**Issue by Francisco6x** [PeWu/topola-viewer#64]
My mother is married to my father.
She has a maiden name and a married name.
Why doesn't Topola-viewer display the married name?
Thanks. Francisco

**Comment by PeWu** [comment]
Topola Viewer displays the name that is set in the `NAME` tag in the GEDCOM file. I see that a custom tag `_MARNM` is used by MyHeritage to store the married name.

I see this as something that could be added in the details panel on the right.

**Comment by chrisdane** [comment]
> A suggestion. Is it possible to enter a test similar to the following in Topola Viewer? If exist _MARNM then NAME = (GIVN + _MARNM)

Yes, please! And if possible with a small word in between, something like `Smith born Walker` or (in German) `Müller geb. Wagner`.

Thanks!
